collection/nischal.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "nischal",
+    "meaning": "Calm",
+    "aliases": [
+        "nichal"
+    ],
+    "translations": {
+        "telugu": "నిశ్చల్"
```

Comment:
Can you please update the translations to use the correct ISO 639-3 code?
List of ISO codes can be found [here](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).